clang/test/Driver/sycl-int-footer.cpp

```diff
@@ -0,0 +1,8 @@
+/// Check compilation tool steps when using the integrated footer
```

Comment:
A general thought that is not related to this PR: would it make sense to move SYCL tests into a separate folder? Potential benefits could be:
1. A tad easier to launch SYCL-specific tests locally (`llvm-lit clang/test/Driver/SYCL/` instead of `llvm-lit clang/test/Driver/sycl-* # Let's hope there are no other namings`)
2. Less clutter in the base directory
3. Given that part of this effort, if undertaken, could be nice to have in the LLORG upstream, it could serve as a precedent/positive example for the community to sort other toolchain-specific tests into subfolders

Reply:
This makes sense to me.  It would also give us an opportunity to break down some of the tests - they have gotten a little unwieldy.

Reply:
I've created https://github.com/intel/llvm/issues/3817 so that we don't lose this thread/can have further discussions there, if needed.